src/app.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
+/* eslint-disable no-console */
 
-// Write your code here
+const readline = require('readline');
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput');
+const { generateRandomNumber } = require('./modules/generateRandomNumber');
+const { getBullsAndCows } = require('./modules/getBullsAndCows');
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const guessNumber = generateRandomNumber();
+let count = 0;
+
+function playGame(question = '') {
+  terminal.question(question, (userInput) => {
+    if (!checkIsValidUserInput(userInput)) {
+      console.log(
+        'It is not valid number! Input the 4 dight number, without duplicates!',
+      );
+      playGame('Try again!');
+    } else {
+      count++;
+
+      const result = getBullsAndCows(userInput, guessNumber);
+
+      console.log(result, '(Try', count, ')');
```

Comment:
Code quality issue: The way you are logging the result is a bit confusing. It would be better to use a template string to make it clearer. For example: `console.log(`${result} (Try ${count})`);`